server/mvc/models/User.js

```diff
@@ -8,29 +8,21 @@ var config,
 
 module.exports = function(){
   var self = this,
-    document = new ThinkyModel({}),
-    exists = false;
+    document = new ThinkyModel({});
 
   this.findByUserId = function(userId){
```

Comment:
Ah, that makes sense.  Agreed, it should not be a "getter", but I am not sure what to name it other than "find".  Not big deal.